### PR TITLE
[OP-480] Add setup instructions

### PR DESCRIPTION
**Pull request by clouths**
Issue: [here](https://github.com/OpenPerpetuum/PerpetuumServer/issues/480)

Add basic setup instructions to help setup the workspace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,65 @@ To see what we are working on acquaint yourself with our:
...
+
+6. (Requires Windows for now) Run the PerpetuumDedicated from steam or directly the exe, install SQL server express (known issues with Win11+, unable to install, use windows 10 to install), initialize the DB when asked at the end.
+
+At this point, take note of the path of the installed server, usually `C:\PerpetuumDedicated`.
+
+## Move/extraction of files
+1. Extract layers archive into the data folder of the PerpetuumDedicated installation
+2. Extract icons from the archive to the `OPResource/lang0000` folder.
+
+Note that the icons archive does not contain icons for new items, the resource server will substitute the default icon instead. The default icon is the blue cube.
```
Comment (aqpanaciy) [inline README.md]: The default icon for new OPP icons is [this](https://github.com/OpenPerpetuum/OPResource/blob/dev/lang0000/icons/default/00000000.png) one.

Comment (aqpanaciy) [reply README.md]: Maybe just remove the last sentence. -> _The default icon is the blue cube._

Comment (clouths) [reply README.md]: Oh I thought it was the blue cube, ok, updating the readme.

**Review by aqpanaciy — APPROVED** [review]
I think that this instruction will help new developers complete the quest to set up a server development environment 😃

**Review by Sellafield — APPROVED** [review]

**Pull request merged**